rust/src/measurements/laplace/float/pseudocode/make_vector_float_laplace_cks20.py

```diff
@@ -0,0 +1,27 @@
+# type: ignore
+def make_vector_float_laplace_cks20(input_domain, input_metric, scale: QO, k):
+    if scale.is_sign_negative():
+        raise ValueError("scale must not be negative")
+    
+    k, relaxation = get_discretization_consts(k)
+
+    # each value in the input is rounded
+    if not relaxation.is_zero():
```

Comment:
```suggestion
    if scale < 0:
        raise ValueError("scale must not be negative")
    
    k, relaxation = get_discretization_consts(k)

    # each value in the input is rounded
    if relaxation != 0:
```

If you wanted the pseudocode to look like typical python, it could be simplified... but maybe you want to keep it closer to Rust idioms? Either approach is fine.